src/meshapi/validation.py

```diff
@@ -61,10 +62,14 @@ def __init__(self, street_address: str, city: str, state: str, zip: int):
                 "text": self.address,
                 "size": 1,
             }
-            nyc_planning_req = requests.get(f"https://geosearch.planninglabs.nyc/v2/search", params=query_params)
+            nyc_planning_req = requests.get(
+                "https://geosearch.planninglabs.nyc/v2/search",
+                params=query_params,
+                timeout=DEFAULT_EXTERNAL_API_TIMEOUT_SECONDS,
+            )
             nyc_planning_resp = json.loads(nyc_planning_req.content.decode("utf-8"))
         except Exception as e:
-            print(f"Got exception querying geosearch.planninglabs.nyc: {e}")
+            logging.error(f"Got exception querying geosearch.planninglabs.nyc: {e}")
```

Comment:
nit: You can actually just do this. The logger can automagically get the exception from the context 
```suggestion
            logging.exception(f"Got exception querying geosearch.planninglabs.nyc")
```